[2024-03-21T15:52:00.559+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: branch_python_operator.check_is_actvive manual__2024-03-21T15:51:59.441513+00:00 [queued]>
[2024-03-21T15:52:00.566+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: branch_python_operator.check_is_actvive manual__2024-03-21T15:51:59.441513+00:00 [queued]>
[2024-03-21T15:52:00.566+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-21T15:52:00.575+0000] {taskinstance.py:2214} INFO - Executing <Task(BranchPythonOperator): check_is_actvive> on 2024-03-21 15:51:59.441513+00:00
[2024-03-21T15:52:00.578+0000] {standard_task_runner.py:60} INFO - Started process 85 to run task
[2024-03-21T15:52:00.580+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'branch_python_operator', 'check_is_actvive', 'manual__2024-03-21T15:51:59.441513+00:00', '--job-id', '225', '--raw', '--subdir', 'DAGS_FOLDER/branch_python_operator.py', '--cfg-path', '/tmp/tmp0dv14ivk']
[2024-03-21T15:52:00.581+0000] {standard_task_runner.py:88} INFO - Job 225: Subtask check_is_actvive
[2024-03-21T15:52:00.679+0000] {task_command.py:423} INFO - Running <TaskInstance: branch_python_operator.check_is_actvive manual__2024-03-21T15:51:59.441513+00:00 [running]> on host 50f748820a54
[2024-03-21T15:52:00.834+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vutt' AIRFLOW_CTX_DAG_ID='branch_python_operator' AIRFLOW_CTX_TASK_ID='check_is_actvive' AIRFLOW_CTX_EXECUTION_DATE='2024-03-21T15:51:59.441513+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-21T15:51:59.441513+00:00'
[2024-03-21T15:52:00.835+0000] {python.py:202} INFO - Done. Returned value was: <Task(PythonOperator): do_task>
[2024-03-21T15:52:00.836+0000] {branch.py:35} INFO - Branch into <Task(PythonOperator): do_task>
[2024-03-21T15:52:00.836+0000] {skipmixin.py:178} INFO - Following branch <Task(PythonOperator): do_task>
[2024-03-21T15:52:00.852+0000] {taskinstance.py:2728} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 234, in execute
    return self.do_branch(context, super().execute(context))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/branch.py", line 36, in do_branch
    self.skip_all_except(context["ti"], branches_to_execute)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/skipmixin.py", line 194, in skip_all_except
    raise AirflowException(
airflow.exceptions.AirflowException: 'branch_task_ids' must be either None, a task ID, or an Iterable of IDs, but got 'PythonOperator'.
[2024-03-21T15:52:00.855+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=branch_python_operator, task_id=check_is_actvive, execution_date=20240321T155159, start_date=20240321T155200, end_date=20240321T155200
[2024-03-21T15:52:00.899+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 225 for task check_is_actvive ('branch_task_ids' must be either None, a task ID, or an Iterable of IDs, but got 'PythonOperator'.; 85)
[2024-03-21T15:52:00.913+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-21T15:52:00.950+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
